app/models/user.rb

```diff
@@ -117,16 +117,15 @@ def can_access_id_card?
   end
 
   def self.from_merged_attrs(existing_user, new_user)
-    # we want to always use the more recent attrs so long as they exist
-    attrs = new_user.attributes.map do |key, val|
-      { key => val.presence || existing_user[key] }
-    end.reduce({}, :merge)
-
     # for loa, we want the higher of the two
+    attrs = new_user.attributes
     attrs[:loa][:current] = [existing_user[:loa][:current], new_user[:loa][:current]].max
```

Comment:
Do we want the updated user to always be the source of the current LOA? Would be nice to remove that as a concern of app and use what the provider gives us.

Reply:
I think it might be worth looking at that as a bigger refactor. This is something that could be easily tested in isolation right now. I would think making the change you describe would require a corresponding F/E change too, unless I'm misunderstanding.